### DISCUSSION TITLE
Want to publish artifact on Jfrog

### DISCUSSION THREAD

**Discussion by RATANAJANGIR** [buildpacks/community#216] [Q&A]
Hi,

I am new on build Pack.
How to build artifact on Jfrog, After running analyze, detect, restore and build lifecycle, I want to publish Artifact package ( JAR OR wheel package) on Jfrog before running export lifecycle. 
Please help me to understand and to resolve this issue.

**Comment by natalieparellano** [comment]
Linking back to pack discussion: https://github.com/buildpacks/pack/issues/1776
